.github/workflows/main.yml

```diff
@@ -642,9 +648,20 @@ jobs:
         run: |
           echo "WREMNANTS_OUTDIR=${WREMNANTS_OUTDIR}" >> $GITHUB_ENV
 
-      - name: clean
+      - name: copy clean plots
         run: |
           echo "xrdcp --parallel 4 -R -f $WREMNANTS_OUTDIR/$LOCAL_WEB_DIR/* root://eosuser.cern.ch//$EOS_DIR/$LOCAL_WEB_DIR"
           xrdcp --parallel 4 -R -f $WREMNANTS_OUTDIR/$LOCAL_WEB_DIR/* root://eosuser.cern.ch//$EOS_DIR/$LOCAL_WEB_DIR
+          echo "Removing temp directory $WREMNANTS_OUTDIR/$LOCAL_WEB_DIR"
+          rm -r $WREMNANTS_OUTDIR/$LOCAL_WEB_DIR
+          
+      - name: save analysis files
+        if: github.event_name != 'pull_request' && github.event.schedule == '0 3 * * 2,4,6'
```

Comment:
Does the first condition add anything?

Reply:
I'm not sure, I have seen it like this at another place and just copied the line